---
description: 詳細については、' Catch ' ブロックに到達していません。 <exception> 同じ ' Try ' ステートメントで上の処理が行われています
title: "'Catch' ブロックに到達しませんでした。'<exception>' は、同じ Try ステートメントで処理しました。"
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: fc52c21f7a319c26b6ca7128c8d2d781db1a58c1
ms.sourcegitcommit: 10e719780594efc781b15295e499c66f316068b8
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 02/14/2021
ms.locfileid: "100473022"
---
# <a name="catch-block-never-reached-exception-handled-above-in-the-same-try-statement"></a><span data-ttu-id="20301-103">'Catch' ブロックに到達しませんでした。'\<exception>' は、同じ Try ステートメントで処理しました。</span><span class="sxs-lookup"><span data-stu-id="20301-103">'Catch' block never reached; \<exception> handled above in the same 'Try' statement</span></span>

<span data-ttu-id="20301-104">先行する `Catch` ブロックで処理されるため、コードの `Try` ブロックには到達できません。</span><span class="sxs-lookup"><span data-stu-id="20301-104">A `Catch` block in the code cannot be reached because it is handled in a preceding `Try` block.</span></span>  
  
<span data-ttu-id="20301-105">既定では、このメッセージは警告です。</span><span class="sxs-lookup"><span data-stu-id="20301-105">By default, this message is a warning.</span></span> <span data-ttu-id="20301-106">警告を非表示にする方法や、警告をエラーとして扱う方法の詳細については、「 [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)」をご覧ください。</span><span class="sxs-lookup"><span data-stu-id="20301-106">For more information about hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>
  
 <span data-ttu-id="20301-107">**エラー ID:** BC42031</span><span class="sxs-lookup"><span data-stu-id="20301-107">**Error ID:** BC42031</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="20301-108">このエラーを解決するには</span><span class="sxs-lookup"><span data-stu-id="20301-108">To correct this error</span></span>  
  
- <span data-ttu-id="20301-109">不要なステートメントを削除します。</span><span class="sxs-lookup"><span data-stu-id="20301-109">Remove the redundant statement.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="20301-110">関連項目</span><span class="sxs-lookup"><span data-stu-id="20301-110">See also</span></span>

- [<span data-ttu-id="20301-111">Try ブロックと Catch ブロックを使用して例外をキャッチする方法</span><span class="sxs-lookup"><span data-stu-id="20301-111">How to use the try/catch block to catch exceptions</span></span>](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [<span data-ttu-id="20301-112">ユーザー フィルター例外ハンドラーの使用</span><span class="sxs-lookup"><span data-stu-id="20301-112">Using User-Filtered Exception Handlers</span></span>](../../standard/exceptions/using-user-filtered-exception-handlers.md)
- [<span data-ttu-id="20301-113">Try...Catch...Finally ステートメント</span><span class="sxs-lookup"><span data-stu-id="20301-113">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
